platforms/core-execution/execution/src/main/java/org/gradle/internal/execution/UnitOfWork.java

```diff
@@ -43,6 +44,8 @@
 
 public interface UnitOfWork extends Describable {
 
+    CachingDisabledReason NOT_WORTH_CACHING = new CachingDisabledReason(CachingDisabledReasonCategory.NOT_CACHEABLE, "Not worth caching.");
```

Comment:
🤔 Should it be static?

Reply:
It's in an interface, this is already static.